repo/content/content_manager_indexes.go

```diff
@@ -79,8 +79,7 @@ func (sm *SharedManager) CompactIndexes(ctx context.Context, opt CompactOptions)
 	return nil
 }
 
-// ParseIndexBlob loads entries in a given index blob and returns them.
-func ParseIndexBlob(ctx context.Context, blobID blob.ID, encrypted gather.Bytes, crypter crypter) ([]Info, error) {
+func ParseIndexBlob(blobID blob.ID, encrypted gather.Bytes, crypter crypter) ([]Info, error) {
```

Comment:
nit: looks like this doc-comment was removed by accident
```suggestion
// ParseIndexBlob loads entries in a given index blob and returns them.
func ParseIndexBlob(blobID blob.ID, encrypted gather.Bytes, crypter crypter) ([]Info, error) {

Reply:
@adowair yes... your suggestion was committed - thank you.